crates/bevy_dogoap/src/macros.rs

```diff
@@ -1,82 +1,62 @@
+//! Conveniece macros
+
+/// Creates a [`Planner`](crate::prelude::Planner) from
+/// - A list of actions
+/// - A list of states
+/// - A list of goals
 #[macro_export]
 macro_rules! create_planner {
     ({
         actions: [$(($action_type:ty, $action:expr)),* $(,)?],
         state: [$($state:expr),* $(,)?],
         goals: [$($goal:expr),* $(,)?],
     }) => {{
-        let actions_map = create_action_map!($(($action_type, $action.clone())),*);
-
-        let components = create_state!($($state.clone()),*);
-
-        let planner = Planner::new(components, vec![$($goal.clone()),*], actions_map);
-
-        let component_entities = ($($state.clone()),*);
-
-        (planner, component_entities)
-    }};
-}
-
-#[macro_export]
-macro_rules! create_action_map {
```

Comment:
Seems this was deleted and merged with `create_planner`? Maybe it's just me, but Rust macros are hairy enough I need to be able to debug them separately and ideally as isolated as possible, and would prefer them to remain separated.